daprdocs/content/en/reference/api/placement_api.md

```diff
@@ -0,0 +1,64 @@
+---
+type: docs
+title: "Placement API reference"
+linkTitle: "Placement API"
+description: "Detailed documentation on the Placement API"
+weight: 1200
+---
+
+Dapr has an HTTP API `/placement/state` for placement service that exposes placement table information. The API is exposed on the sidecar on the same port as the healthz. This is an unauthenticated endpoint, and is disabled by default. 
+
+You need to set `DAPR_PLACEMENT_METADATA_ENABLED` environment or `metadata-enabled` command line args to true to enable it. If you are using helm, set `dapr_placement.metadataEnabled` to true.
```

Comment:
This need an update into the https://docs.dapr.io/reference/environment/ topic, with the Dapr sidecar as the target.

Reply:
One more thing, the environment variable is for Dapr placement, so the target would be Dapr placement.

Reply:
Pushed this update - thanks!